build/cpp/kvstore/src/kvstore.cc

```diff
@@ -1,6 +1,41 @@
 #include "kvstore.h"
 
-#include <iostream>
+// key value store constructor
+kvstore::KVStore::KVStore(const std::string &store) {
+  store_ = "data/" + store;
```

Comment:
Would recommend against this because now all the files must be within the data folder

Reply:
"data/" was removed.